data/announcements/scheduled.yaml

```diff
@@ -116,3 +116,16 @@ announcements:
   message: |
     5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
     May 16 - 20, 2022.
+
+- name: Kubecon 2022 NA
+  startTime: 2022-08-22T00:00:00 #Added in https://github.com/kubernetes/website/pull/36107
```

Comment:
This seems too early. We usually only turn the banner on around 1 week before the conference starts.

Reply:
oh...I see.
Would extend the start date to 17th of October, 2022